src/solver/hydro/management/management.cpp

```diff
@@ -281,15 +281,15 @@ void HydroManagement::checkWeeklyMinGeneration(uint tsIndex, Data::Area& area) c
     }
 }
 
-void HydroManagement::checkHourlyMinGeneration(uint tsIndex, Data::Area& area) const
+void HydroManagement::checkHourlyMinMaxGeneration(uint tsIndex, Data::Area& area) const
```

Comment:
For each MC problem, TS-s for min and max can be different by new specification, so this function must have information about TS-s indexes  of min and max matrix in order to compare it. These indexes are linked to MC year (problem) in scenario builder, so values of indexes will be extract from some structure that represents incident MC year. 

With this extract ts indexes, but this structure must have additional element that will represent TS-s of max generation, because as it is mentioned, TS's could have different indexes.
(*NumeroChroniquesTireesParPays[numSpace][k]).Hydraulique;